build.gradle

```diff
@@ -7,10 +7,11 @@ buildscript {
         jcenter()
         maven { url 'https://maven.fabric.io/public' }
         maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+        maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
Why do we need this?

Reply:
this url is added for the coverall upgrade